gax/src/test/java/com/google/api/gax/rpc/OperationCallableImplTest.java

```diff
@@ -721,7 +729,9 @@ public void testFutureCancelImmediately() throws Exception {
     OperationFuture<Color, Currency> future =
         callable.futureCall(2, FakeCallContext.createDefault());
 
-    CancellationHelpers.cancelInThreadAfterLatchCountDown(future, retryScheduledLatch);
+    while (!future.cancel(true) && !future.isDone()) {
```

Comment:
Why is this change necessary?

Reply:
This performs the cancelation in same thread, which runs the test (i.e. has higher chance of being called earlier than the `CancelationHelpers` version, which calls the cancelation in a separate thread). The test is called `testCancelImmediately`, so the idea is to cancel as quickly as possible.